vnext/Shared/OInstance.cpp

```diff
@@ -339,6 +340,10 @@ InstanceImpl::InstanceImpl(
       Microsoft::ReactNative::PackagerConnection::CreateOrReusePackagerConnection(*m_devSettings);
     }
 
+    if (Microsoft::React::GetRuntimeOptionBool("ForceSystemChakra")) {
```

Comment:
>GetRuntimeOptionBool [](start = 26, length = 20)

Why do we need this here? We can just use m_devSettings->jsiEngineOverride, no? Or is it for global FG?

Reply:
It's a global FG, but setting it in RNHost may not be enough.

There are several spots in RNW that need to access this setting, or the correct code won't be called.